app.js

```diff
@@ -77,9 +69,24 @@ function getRandomQuestion(){
     return JSON.parse(randomQuestion);
 }
 
-function checkTheAnswer(userAnswer){
+function checkTheAnswer(userAnswer, socket){
+
     if(question.answer.toLowerCase() === userAnswer.toLowerCase()){
+        users.map(function(user){
```

Comment:
you are not using `map` exactly right.

map takes one array and returns another.

`var newUsers = users.map(someFn)`.

If you don't want to get new result just use `forEach` instead of map.